src/components/sidebar/index.tsx

```diff
@@ -108,15 +114,39 @@ const Sidebar = ({
   const [filterValue, setFilterValue] = useState('')
   const filteredMenuItems = getFilteredMenuItems(itemsWithMetadata, filterValue)
 
-  let backToLink
-  if (backToLinkProps) {
+  let backToElement
+  if (levelButtonProps) {
+    const { iconPosition, onClick, text } = levelButtonProps
+    backToElement = (
+      <Button
+        className={classNames(s.levelButton, {
+          [s.levelButtonWithLeadingIcon]: iconPosition === 'leading',
+          [s.levelButtonWithTrailingIcon]: iconPosition === 'trailing',
+          [s.levelButtonBeforeFilterInput]: showFilterInput,
+          [s.levelButtonBeforeHeading]: !showFilterInput && !visuallyHideTitle,
+          [s.levelButtonBeforeNavItem]: !showFilterInput && visuallyHideTitle,
+        })}
```

Comment:
The spacing below this element is different depending on what element comes after it. Some examples [here](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Dev-Portal?node-id=7502%3A69101) and [here](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Dev-Portal?node-id=7984%3A83409).

Reply:
Removed, no longer relevant! ✨